src/ffi/pylifecycle.rs

```diff
@@ -7,6 +7,7 @@ extern "C" {
     pub fn Py_Initialize();
     pub fn Py_InitializeEx(arg1: c_int);
     pub fn Py_Finalize();
+    #[cfg(any(not(Py_LIMITED_API), Py_3_6))]
```

Comment:
This isn't necessary - `Py_3_6` is always set, because we don't support older Python versions than 3.6.

```suggestion
```

Reply:
Doesn't know the Python version 3.6 support before. Now got it and fixed.